website/src/routes/pages.js

```diff
@@ -5,6 +5,8 @@ var _ = require('lodash');
 var locals = require('../middlewares/locals');
 var i18n = require('../libs/i18n');
 
+var userCount = '1,000,000';
```

Comment:
hmm @crookedneighbor is const supported in the api v2 environment? it's still on node 0.10